src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Hooks.java

```diff
@@ -1,147 +1,166 @@
-        options.setExperimentalOption("prefs", chromePrefs);
-        chromePrefs.put("safebrowsing.enabled", "true");
-        options.addArguments("--safebrowsing-disable-download-protection");
-        options.addArguments("disable-popup-blocking");
-
-        DesiredCapabilities cap = DesiredCapabilities.chrome();
-        cap.setCapability(ChromeOptions.CAPABILITY, chromeOptionsMap);
-        cap.setCapability(CapabilityType.ACCEPT_SSL_CERTS, true);
-        cap.setCapability(ChromeOptions.CAPABILITY, options);
-
-        screenshotPath = System.getProperty("user.dir") + File.separator + "test-output";
-        driver = new EventFiringWebDriver(new ChromeDriver(cap));
-        driver.manage().timeouts().implicitlyWait(DELAY, TimeUnit.SECONDS);
-        WebDriverEventListener errorListener = new AbstractWebDriverEventListener() {
-            @Override
-            public void onException(final Throwable throwable, final WebDriver activeDriver) {
-                try {
-                    takeScreenshot();
-                } catch (Exception e) {
-                    e.printStackTrace();
-                }
-            }
-        };
-        driver.register(errorListener);
-    }
-
-    /**
-     * Close browser windows and terminate WebDriver session.
-     */
-    @After("~@RegressionAPI")
-    public void afterMethod() {
-        driver.quit();
-    }
-
-    public static EventFiringWebDriver getDriver() {
-        return driver;
-    }
-
-    public static File getDownloadDirectory() {
-        return dir;
-    }
-
-    /**
-     * Take a screenshot.
-     * @param hash a string that will be inserted into the filename to avoid overwriting images.
-     * All "non-word characters" will be replaced with "_"
-     * @throws Exception if there is an exception
-     */
-    public static void takeScreenshot(final String hash) throws Exception {
-        File scrFile = ((TakesScreenshot) getDriver()).getScreenshotAs(OutputType.FILE);
-        File outFile = new File(screenshotPath + File.separator + "failed-test-" + hash.replaceAll("\\W+", "_") + ".png");
-        FileUtils.copyFile(scrFile, outFile);
-    }
-
-    /**
-     * Take a screenshot. Generates hash based on current time.
-     * @throws Exception if screenshot cannot be created
-     */
-    public static void takeScreenshot() throws Exception {
-        String hash = String.valueOf(new Date().getTime());
-        takeScreenshot(hash);
-    }
-    
-
-}
+		 */
+
+		if(System.getenv("browser")!=null && !System.getenv("browser").isEmpty()){
+			browser = System.getenv("browser");
+			System.setProperty("webdriver.gecko.driver", System.getProperty("PathtoGeckodriver") + File.separator + "geckodriver");
+
+		}else{
+			fis = new FileInputStream(
+					System.getProperty("user.dir") + File.separator+"src"
+							+File.separator+"test"+File.separator+"resources"
+							+File.separator+"Config.properties");
+			config.load(fis);
```

Comment:
Why is this config file loaded only if there's no browser defined?

Reply:
This else part is handling while running locally and if part is handling input from jenkins job(this is set up on my fork-jenkins).
So when running locallly, we need to load config file otherwise no need.